docs/pages/blog/lab-tree-view-to-mui-x.md

```diff
@@ -0,0 +1,71 @@
+---
+title: The Tree View is moving to MUI X
+description: Migrate to the new package to start building with our powerful Tree View, now part of MUI X. Previously released MIT components will stay MIT.
+date: 2023-08-11T00:00:00.000Z
+authors: ['flaviendelangle']
+tags: ['MUI X', 'News']
+card: true
```

Comment:
```suggestion
card: false
```

unless you will add card later?

Reply:
Fixed :+1:

Reply:
It's still broken https://robolly.com/open-graph-preview/?url=https://mui.com/blog/lab-tree-view-to-mui-x/

<img width="462" alt="Screenshot 2023-09-06 at 20 56 52" src="https://github.com/mui/material-ui/assets/3165635/205cfdc1-792f-412d-8cff-af1a3e7a6c6a">

It's fixed in #38834.